### PR TITLE
[TASK] Add missing page.meta attribute sub-property

### PR DESCRIPTION
**Pull request by brotkrueml**
The more generic `attribute` sub-property can be used to set the attribute to any arbitrary value. An example is already available for the Open Graph meta tags. Therefore, it is more generic than the `httpEquivalent` sub-property.

Releases: main, 12.4

### CODE REVIEW

**Comment by brotkrueml** [comment]
Waiting for #1336 to be merged as it may conflict with this one.

**Comment by linawolf** [comment]
Where do you have the information from? Source?

**Comment by brotkrueml** [comment]
Real world.

Look at the examples at the end of the page, there `attribute` is used.

It took me quite some time to get `og:image:width` working (it only works with `attribute = property` given - without it, an error occurs).

**Comment by brotkrueml** [comment]
@linawolf Can we proceed here?

**Comment by linawolf** [comment]
I am not blocking this, but someone who knows more about it would have to review

**Comment by brotkrueml** [comment]
As there is no interest in this PR, closing it.

**Comment by garvinhicking** [comment]
I nust stumbled across this. Would you like to reopen? I'd review :-)

**Comment by brotkrueml** [comment]
I can't as I deleted the branch. Feel free to add it again.

File: Documentation/TopLevelObjects/Page/Index.rst [hunk 1]
```diff
@@ -591,9 +591,14 @@ Properties
 
         For each key the following sub-properties are available:
 
+        `attribute`
+            Sets the attribute for the meta tag. If it is not defined, the
+            default :html:`name` is used.
```
Comment (garvinhicking) [inline Documentation/TopLevelObjects/Page/Index.rst:596]: Maybe we could be even more verbose here along the lines of:

The :html:<meta> Tag usually uses attributes like :html:<meta name="..."> (HTML5), but :html:<meta property="..."> is also valid for RDFa metadata like specifyinf OpenGraph properties. By specifying the 'attribute' key, the used attribute can be set to 'name' or 'property', defaulting to 'name'

But current version is already an improvement and I'd be happy with it, too!

**Review by garvinhicking — APPROVED** [review]

File: Documentation/TopLevelObjects/Page/Index.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/TopLevelObjects/Page/Index.rst:596]: Please make a follow-up. I won't work on this PR anymore.

Comment (garvinhicking) [reply Documentation/TopLevelObjects/Page/Index.rst:596]: Hm.

**Comment by garvinhicking** [comment]
Probably not going to backport this manually.

**Comment by linawolf** [comment]
Manually backported

**Pull request merged**